lib/nbdkit.ml

```diff
@@ -27,13 +27,14 @@ open Unix_utils
 open Utils
 
 let is_installed =
-  let test = lazy (Sys.command "nbdkit --version >/dev/null 2>&1" = 0) in
-  fun () -> Lazy.force test
+  let cmd = sprintf "%s --version >/dev/null 2>&1" Config.nbdkit in
+  let test = lazy (Sys.command cmd = 0) in
+  Lazy.force test
```

Comment:
This just sets `is_installed` to `test`.  You need to keep the `fun () -> Lazy.force test` in there.  The rest of this hunk is fine.

Reply:
You're right, I'll fix it ASAP